src/map/skill.cpp

```diff
@@ -1965,6 +1965,11 @@ int32 skill_additional_effect( struct block_list* src, struct block_list *bl, ui
 	case SO_POISON_BUSTER:
 		sc_start(src,bl, SC_POISON, 5 * skill_lv, skill_lv, skill_get_time(skill_id, skill_lv));
 		break;
+	case SO_CLOUD_KILL: {
+		sc_type sct = skill_get_sc(skill_id);
+		sc_start(src, bl, sct, 100, skill_lv, skill_get_time2(skill_id, skill_lv));
+ 		break;
+	}
```

Comment:
```suggestion
	case SO_CLOUD_KILL:
		sc_start(src, bl, skill_get_sc(skill_id), 100, skill_lv, skill_get_time2(skill_id, skill_lv));
 		break;
```
Was there meant to be a safety check for `sct`?

Reply:
I forgot to update this part after updating the status in status_db.yml